db/migrate/20161127084744_remove_featuredfrom_skins.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveFeaturedfromSkins < ActiveRecord::Migration[4.2]
```

Comment:
Just a note to update the date on this before it's merged so there won't be any issues with the updated schema (#3643).